packages/vkui/src/components/ModalRoot/ModalRoot.test.tsx

```diff
@@ -121,4 +121,25 @@ describe.each([
       });
     }
   });
+
+  it('check modal mask with fast modal change', async () => {
+    const onClose = jest.fn();
+    const modals = [
+      <ModalPage id="m" key="m" />,
+      <ModalPage id="other" key="o" onClose={onClose} />,
+    ];
+
+    const h = render(<ModalRoot activeModal="m">{modals}</ModalRoot>);
+    runAllTimers();
+    h.rerender(<ModalRoot activeModal={null}>{modals}</ModalRoot>);
+    h.rerender(<ModalRoot activeModal="other">{modals}</ModalRoot>);
+    runAllTimers();
+
+    // check if mask is present
+    expect((document.querySelector('.vkuiModalRoot__mask') as HTMLElement).style.opacity).toBe('1');
```

Comment:
```suggestion
    expect((document.querySelector<HTMLElement>('.vkuiModalRoot__mask')).style.opacity).toBe('1');
```

вижу в это тестовом файле изначально селекторы не надёжно берутся, лучше из `import styles from './ModalRoot.module.css;' вытаскивать:

```suggestion
    expect((document.querySelector<HTMLElement>(styles['.ModalRoot__mask'])).style.opacity).toBe('1');
```

Reply:
Поправила везде в этом файлике х)